.idea/modules/Machine-shop-simulator_test.iml

```diff
@@ -3,7 +3,9 @@
   <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_8">
     <output-test url="file://$MODULE_DIR$/../../out/test/classes" />
     <exclude-output />
-    <content url="file://$MODULE_DIR$/../../src/test" />
+    <content url="file://$MODULE_DIR$/../../src/test">
```

Comment:
these iml and xml changes should not be uploaded.  This could complicate someones gradle settings, making the project not run